extensions/wikia/ImageReview/PromoteImageReview.setup.php

```diff
@@ -30,6 +30,7 @@
 if( function_exists('extAddBatchTask') ) {
 	extAddBatchTask($dir . "PromoteImageReviewTask.php", "promoteimagereview", "PromoteImageReviewTask");
 }
+//Possibly this line can be removed (as the usage new Wikia\\Tasks\\Tasks\\PromoteImageReviewTask was removed)

```

Comment:
I consulted that with Mix and Paweł and Sus is already working on retiring special promote, so they'll handle that as part of that effort (for which I presume they already have a ticket)